action.yml

```diff
@@ -27,7 +27,10 @@ runs:
       id: action
       shell: powershell
       run: |
-        $secure_password = ConvertTo-SecureString -String '${{ inputs.service-account-password }}' -AsPlainText -Force
+        $password_string = @'
+        ${{ inputs.service-account-password }}
+        '@
+        $secure_password = ConvertTo-SecureString -String $password_string -AsPlainText -Force
```

Comment:
I have a question about this...it seems like we keep changing the implementation of how the password string is handled... how do we prevent us needing to keep changing that?